### PR TITLE
ilm: Add doc to the complete mc guide

### PR DESCRIPTION
**Pull request by vadmeste**
Adding missing ilm command in the doc

### CODE REVIEW

File: docs/minio-client-complete-guide.md [hunk 1]
```diff
@@ -1216,6 +1216,48 @@ mc event add play/andoria arn:minio:sqs:us-east-1:1:your-queue --prefix photos/
...
+*Example: List the lifecycle management rules (expiration date/days fields)*
+
+```
+mc ilm list --expiry myminio/testbucket
+   ID    | Prefix | Enabled | Expiry |  Date/Days   | Transition | Date/Days | Storage-Class | Tags
+---------|--------|---------|--------|--------------|------------|-----------|---------------|------
+ Devices |  dev/  |    ✓    |   ✓    | 17 Sep 2020  |     ✗      |           |               |
+---------|--------|---------|--------|--------------|------------|-----------|---------------|------
+```
+
```
Comment (klauspost) [inline docs/minio-client-complete-guide.md:1248]: Explain what non-obvious fields are or link.

File: docs/minio-client-complete-guide.md [hunk 2]
```diff
@@ -1216,6 +1216,48 @@ mc event add play/andoria arn:minio:sqs:us-east-1:1:your-queue --prefix photos/
...
+*Example: Add rule for testbucket on s3*
+```
+mc ilm add --id "Devices" --prefix "dev/" --expiry-date "2020-09-17" --transition-date "2020-05-01" --storage-class "GLACIER" s3/testbucket
```
Comment (klauspost) [inline docs/minio-client-complete-guide.md:1251]: I don't suppose MinIO supports storage class glacier? Could we add an example that should work?

File: docs/minio-client-complete-guide.md [hunk 3]
```diff
@@ -1216,6 +1216,48 @@ mc event add play/andoria arn:minio:sqs:us-east-1:1:your-queue --prefix photos/
...
+
+```
+
+*Example: List the lifecycle management rules (expiration date/days fields)*
+
+```
+mc ilm list --expiry myminio/testbucket
+   ID    | Prefix | Enabled | Expiry |  Date/Days   | Transition | Date/Days | Storage-Class | Tags
+---------|--------|---------|--------|--------------|------------|-----------|---------------|------
+ Devices |  dev/  |    ✓    |   ✓    | 17 Sep 2020  |     ✗      |           |               |
```
Comment (klauspost) [inline docs/minio-client-complete-guide.md:1245]:
```suggestion
 Devices |  dev/  |    ✓    |   ✓   | 17 Sep 2020  |     ✗      |           |               |
```

File: docs/minio-client-complete-guide.md (hunk 2 above)
Comment (harshavardhana) [reply docs/minio-client-complete-guide.md:1251]: Agreed we should generally limit talking AWS S3 in our docs, examples can have an affinity towards MinIO

**Pull request merged**